index.html

```diff
@@ -1806,30 +1809,49 @@
                   </ol>
                 </li>
                 <li>If <var>previousAvailability</var> is not equal to
-                <var>newAvailability</var>, then <a>queue a task</a> to run the
-                following steps:
+                <var>newAvailability</var>, then run the following steps:
                   <ol>
-                    <li>Set <var>A</var>'s <a>value</a> property to
-                    <var>newAvailability</var>.
+                    <li>If <var>A</var> has already been exposed to ECMAScript
```

Comment:
I believe something like this is needed. Let me try to clarify why:

Given a fresh browser session (no previous call to the Presentation API and no previous run of the monitoring algorithm) and a `PresentationRequest` `request`, if there is an available presentation display for `request`, I'd like two things when I call `getAvailability`:

1. to get a `PresentationAvailability` object whose `value` property is `true`
2. **not to** receive a `change` event unless availability really changes

In other words, I would expect the following code not to report the error messages:

```js
request.getAvailability().then(function (A) {
  if (A.value !== true) {
    console.error('The available presentation display was not found!');
  }
  A.onchange = function (evt) {
    // The first "change" event received should change the value to "false"
    if (A.value === true) {
      console.error('Change event received but value did not change!');
    }
    A.onchange = null;
  };
});
```

If we do not make the "exposed" distinction, the monitoring algorithm will always "queue a task" to change the value and fire a "change" event. That task will run **after** the `getAvailability` promise is resolved (promise resolution is a micro-task, and has priority over regular tasks), and thus `A.value` will be `false`. Also, since that task gets run after the `getAvailability` promise is resolved, the JS code will have had time to attach to the `change` event, and the task is no longer a no-op: the event handler will be able to catch that `change` event, which we do not want because we're just setting the initial value.

That task is useful for all other `PresentationAvailability` objects in the set, meaning those that exist due to previous calls to `getAvailability`.

I'm not crazy about "exposed to ECMAScript code". Another way to do this would be to track (A, availabilityUrls, initialized) tuples with `initialized` set to `false` initially. The algorithm would directly set `value` and set this flag to `true` if it is not set, and queue a task to change the value otherwise. I was trying to avoid introducing new variables. What do you think?